src/PhotometryFit.cc

```diff
@@ -68,7 +67,7 @@ void PhotometryFit::LSDerivatives(const CcdImage &ccdImage, TripletList &triplet
         // tweak the measurement errors
         double sigma = measuredStar.eflux;
 #ifdef FUTURE
-        TweakPhotomMeasurementErrors(inPos, measuredStar, _posError);
+        TweakPhotomMeasurementErrors(inPos, measuredStar, _fluxError);
```

Comment:
Ah, ok. It seemed odd that you were adding a reference to `_fluxError` here when you were removing it everywhere else.